src/lib/y2network/config.rb

```diff
@@ -237,7 +237,7 @@ def configured_interface?(iface_name)
     #   to be modified or deleted if `connection_config` is deleted or renamed
     def connections_to_modify(connection_config)
       result = []
-      bond_bridge = connection_config.find_master(connections)
+      bond_bridge = connection_config.find_parent(connections)
```

Comment:
Please include the terms _bridge_ and _parent_ in the terms summary.
Maybe some of them are duplicate terms that could be unified, now or later.

Reply:
bridge and parent are not that related at this place. Here the meaning is bond or bridge. Or vice versa, if the connection has a parent then it is a bond or a bridge device.